src/Components/ProgramsPage.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, FlatList } from 'react-native';
+import { connect } from 'react-redux';
+
+import theme from '../styles/theme.style';
+import Program from './programs/Program';
+
+export const ProgramsPage = props => {
+  const { programs, navigation } = props;
+
+  return (
+    <View style={styles.programPage}>
+      <FlatList
+        data={programs}
+        keyExtractor={item => item.id.toString()}
+        renderItem={({ item }) => <Program program={item} navigation={navigation} />}
+      />
+    </View>
+  );
+};
```

Comment:
If I remember well, we  decided to keep a link to a full trainings list as well, below the predefined program.

Reply:
I have no memory of this 😱

I can bring it back and add it to the home page

Reply:
Adélie is right. We want to keep the training list below the predefined program

Reply:
It's back in the game

Reply:
I am not 100% sure about this but should we want to keep it the new button you have added on the homage into the 'training programs' page ?
![pagetrainingmartin](https://user-images.githubusercontent.com/56125567/80867985-d766a480-8c97-11ea-83d8-a48f35e0a8e2.png)

Here is what I've added in mind

Reply:
I agree with Moby, I think it is quite confusing to have both 'Trainings' and 'Training programs' on homepage (though maybe not so import for a V0). What do you think Martin ?

Reply:
Yep, I'll move it to the programs page as Moby suggested above

Reply:
I just tweaked it a bit because it did'nt look like a cta

Looks like this (probably not perfect but i think it's ok enough for now?)

<img width="468" alt="Screenshot 2020-05-02 at 21 14 05" src="https://user-images.githubusercontent.com/3810795/80873566-dfcfd700-8cb9-11ea-8f73-57020d472fd7.png">